CesiumRasterOverlays/src/RasterOverlayUtilities.cpp

```diff
@@ -403,6 +403,89 @@ bool isSouthChild(CesiumGeometry::UpsampledQuadtreeNode childID) noexcept {
 
 void copyImages(const Model& parentModel, Model& result);
 void copyMetadataTables(const Model& parentModel, Model& result);
+
+/**
+ * @brief Helper struct for working with non-indexed triangles. Returns either
+ * the indices from an index accessor view, or generates new indices.
+ */
+template <typename TIndex> struct IndicesViewRemapper {
+  IndicesViewRemapper(
+      const Model& model,
+      const MeshPrimitive& primitive,
+      int32_t primitiveIndices,
+      int64_t numVertices)
+      : accessorView(std::nullopt),
+        indicesCount(0),
+        primitiveMode(primitive.mode) {
+    AccessorView<TIndex> view(model, primitiveIndices);
+    if (view.status() == AccessorViewStatus::Valid) {
```

Comment:
The logic here appears to be that if the indices accessor is invalid for any reason, the model is treated as if it's non-indexed. I think it would be better to only treat the model as non-indexed when `primitiveIndices` is less than zero, and pass through any other problems as actual errors.